src/core/usecase/transform_to_pre_paid.py

```
@@ -35,7 +35,7 @@ def __init__(
 
     def transform_to_pre_paid(self, external_id: UUID):
         account = self.reading_account.by_external_id(external_id)
-        if account.type is AccountType.POST_PAID:
+        if account.subscription.type is AccountType.POST_PAID_MONTH:
             raise SystemError("This account already is PostPaid")
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)